websphere-liberty/8.5.5/developer/javaee7/README.md

```diff
@@ -0,0 +1,7 @@
+# WebSphere Application Server Developer Edition V8.5.5 Liberty Profile image for Docker
+
+The [Dockerfile](Dockerfile) in this directory is used to build the `websphere-liberty:javaee7` image on [Docker Hub](https://registry.hub.docker.com/_/websphere-liberty/). The image contains IBM WebSphere Application Server Developer Edition V8.5.5 and an IBM Java Runtime Environment.
```

Comment:
In each of these READMEs could you say something about the features that the image contains (just in case it isn't obvious from the name!).